src/test/java/org/intellij/sdk/codesync/clients/ClientUtilsTest.kt

```diff
@@ -0,0 +1,19 @@
+package org.intellij.sdk.codesync.clients
+
+import org.intellij.sdk.codesync.Constants
+import org.intellij.sdk.codesync.clients.ClientUtils
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import kotlin.test.assertNotNull
+import kotlin.test.assertNull
+
+class ClientUtilsTest {
+
+    @Test
+    fun sendPost(){
+        val correctAPI = String.format("%s/users?&source=%s&v=%s", "https://api.codesync.com/v1", "intellij", "unknown")
+        assertNotNull(ClientUtils.sendPost(correctAPI, "ACCESS_TOKEN"))
```

Comment:
Should we assert 401 status code here? Since you are providing invalid access token?